### PR TITLE
Push arlas jars in mvn

### LINKED ISSUE #485: Push arlas jars in maven repos

### LINKED ISSUE #486: Generate a Java client api and push it in maven repo

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
Fix #485

### CODE REVIEW

**Comment by sfalquier** [comment]
Maybe should we close this PR since we found a better way to do the job?

**Comment by MohamedHamouGisaia** [comment]
@sfalquier I made the modifications we discussed in this branch/PR

**Comment by MohamedHamouGisaia** [comment]
@alainbodiguel

File: release.sh [hunk 1]
```diff
@@ -162,6 +168,10 @@ mkdir -p target/tmp || echo "target/tmp exists"
 i=1; until curl -XGET http://${DOCKER_IP}:19999/arlas/swagger.json -o target/tmp/swagger.json; do if [ $i -lt 60 ]; then sleep 1; else break; fi; i=$(($i + 1)); done
 i=1; until curl -XGET http://${DOCKER_IP}:19999/arlas/swagger.yaml -o target/tmp/swagger.yaml; do if [ $i -lt 60 ]; then sleep 1; else break; fi; i=$(($i + 1)); done
 
+mkdir -p arlas-spec
```
Comment (sfalquier) [inline release.sh]: Maybe this folder would better be called `openapi` (no need to prefix with `arlas-` since it is located in arlas-server repository).

Comment (MohamedHamouGisaia) [reply release.sh]: done :)

**Pull request merged**